src/RscChart.tsx

```diff
@@ -163,7 +178,9 @@ export const RscChart = forwardRef<ChartHandle, RscChartProps>(
 			padding
 		);
 
-		const tooltipConfig: TooltipOptions = { theme: colorScheme };
```

Comment:
Could we have a comment here for why the `useMemo`?  Correct me if I'm wrong, but I think colorScheme is just a prop we pass down, so we shouldn't need to cache it here